tools/rna_tools/locarna/locarna.xml

```diff
@@ -1,92 +0,0 @@
-<tool id="locarna" name="locarna" version="0.1.0">
-    <description></description>
-    <requirements>
-    <requirement type="package" version="x.y.z">locarna</requirement>
```

Comment:
Hi @bgruening, I don't think that this was ever working properly -- also I don't find it in the toolshed. The old wrapper would be completely replaced by the new locarna_multiple.

I don't completely see the consequences of removing the old locarna. If you think we should rather not remove "locarna", I am fine with deprecating it (than we can add a reference to the new wrapper).
(Interestingly, there is a mlocarna wrapper in graphclust as well -- I didn't realize this so far.)